src/Domain/Creators/SphereTimeDependentMaps.hpp

```diff
@@ -95,16 +147,35 @@ struct TimeDependentMapOptions {
     using group = ShapeMapOptions;
   };
 
-  using options = tmpl::list<InitialTime, SizeMapInitialValues, ShapeMapLMax>;
+  struct ShapeMapInitialValues {
+    static std::string name() { return "InitialValues"; }
+    // Variants require two templates, but only one is implemented right now.
+    // Rather than doing Auto<KSFBL> and then having to edit the input file
+    // syntax again later on, just add in a not implemented option for now. This
+    // will be added in the future.
+    using optional_type =
+        std::variant<KerrSchildFromBoyerLindquist, FromH5File>;
+    using type = Options::Auto<optional_type, Spherical>;
+    static constexpr Options::String help = {
+        "Initial Ylm coefficients for the shape map. Specify 'Spherical' for "
+        "all coefficients to be initialized to zero."};
+    using group = ShapeMapOptions;
+  };
+
+  using options = tmpl::list<InitialTime, SizeMapInitialValues, ShapeMapLMax,
+                             ShapeMapInitialValues>;
```

Comment:
Can we share the logic in this file with the BCO domains? Would be nice, because they need almost the same logic. Doesn't have to happen in this PR.

Reply:
We might be able to. Not sure what to do about the maps though because currently this assumes size/shape to distorted and then identity to inertial. The combinations of maps are more complicated in the BCO domain. I'd have to think about it. So yeah, definitely for a follow up PR if at all